docs/source/maintenance/restarting_node.rst

```diff
@@ -45,12 +45,6 @@ If you want to reuse WSV state, start Iroha with `--reuse_state` flag.
 Given this flag, Iroha will not reset or overwrite the state database if it fails to start for whatever reason.
 
 
-Enabling WSV Reuse
```

Comment:
why has it got removed?

Reply:
There was a duplication (see above or below — same text, who knows how it got there but needed to be removed)